### ISSUE TITLE
Export functionality for Survey JS

### ISSUE THREAD

**Issue by matof-who** [HeRAMS-WHO/herams-backlog#6]

**Comment by SamMousa** [comment]
Question types
- [ ] Single choice
- [ ] Multiple choice
- [ ] Ranking?
- [ ] Open text

**Comment by matof-who** [comment]
HI @SamMousa 
I've analyzed the different question types here: https://extranet.who.int/confluence/x/woBjB (please let me know if you can't access)
There a couple of tricky ones: dynamic lists, dynamic matrix and ranking

We should compare how SurveyJS manages the export for those ones ... do we have a SurveyJS environment where I could create a dummy survey and export to compare ?

**Comment by SamMousa** [comment]
SurveyJS does not do an export, we have to build that ourselves. So there is no reference export (yet)

**Comment by SamMousa** [comment]
Note for self:
- Export requires us to export localizable texts (like facility name) as individual variables
- Currently we can only set response / facility list visibility on the survey question level
- Need a new solution to only show the name in the current locale:


![Image](https://user-images.githubusercontent.com/547021/210574639-dd6d14fa-b7af-4811-abec-0fc18001e878.png)

**Comment by SamMousa** [comment]
@matof-who just to confirm: the format of this export is unrelated to the format of the import, right?
Ie we need not be able to map string labels during importing back to their underlying values. Or support multiple locales during data imports (other than localized values)

**Comment by matof-who** [comment]
@SamMousa , yes, that's correct. Export and import format are not expected to be identical
Also, I've added to the description the requirement to include the byte order mark to the export file
